sonarts-core/tests/rules/usePrimitiveReturnTypeRule/usePrimitiveReturnTypeRule.lint.ts

```diff
@@ -0,0 +1,37 @@
+// OK, doesn't have `any`
+function foo() {
+  return 1;
+}
+
+function bar(): any {
+  //            ^^^ {{Remove this return type or change it to a more specific.}}
+  return 1;
+}
+
+// OK, returns different primitive types
+function baz(x: any): any {
+  if (x) {
+    return 1;
+  } else {
+    return "baz";
+  }
+}
+
+// OK, returns union type
+function ternary(x: any): any {
+  const y = x ? 1 : 2;
+  return y;
+}
+
+// OK, returns non-primitive type
+function createObject(): any {
+  return { foo: 1, bar: null };
+}
+function useObject() {
```

Comment:
@stas-vilchik what's the purpose of this function?

Reply:
none, I was using it for my tests and forgot to remove